public/js/index.js

```diff
@@ -17,22 +32,22 @@ const createWidget = function (objWidget, objSocket) {
   $(widgetContent).appendTo(widgetContainer)
   // store the data for the widget WITH the widget
   $(widgetContainer).data('widget', objWidget)
-  // add the widgets to the page widget container
   // widget types are uppercase to avoid conflicts with jquery ui widget names
-  $(widgetContainer)[objWidget.type.toUpperCase()]({ ...objWidget, socket: objSocket }).appendTo('#widgets').position({
-    ...objWidget.position,
-    of: '#widgets',
-    collision: 'fit'
-  }).draggable({
-    handle: '.widget-header',
-    snap: true,
-    stop: function (event, ui) {
-      /*
-      const widgetId = $(this).data('widget-id')
-      const widgetPosition = $(this).position()
-      console.log(widgetId, widgetPosition)
-      */
-    }
+  $(widgetContainer)[objWidget.type.toUpperCase()](
+      { ...objWidget, socket: objSocket }
+    ).appendTo(
+      '#widgets'
+    ).draggable(
+      {
+      handle: '.widget-header',
+      snap: true,
+      stop: function (event, ui) {
+        /*
```

Comment:
Do you truly want an empty function here?

Reply:
I did. I debated removing it but it took effort to find the properties I wanted for the event I dont want to lose